src/test/java/gov/usgs/cida/pubs/webservice/DeletedPublicationMvcServiceIT.java

```diff
@@ -38,13 +39,20 @@
 public class DeletedPublicationMvcServiceIT extends BaseIT {
 
 	@Autowired
+	private WebApplicationContext webApplicationContext;
+
 	private MockMvc mockMvc;
 
+	@Before
+	public void setUp() {
+		mockMvc = MockMvcBuilders.webAppContextSetup(webApplicationContext).build();
+	}
+
 	@Test
 	public void getAllNoPaging() throws Exception {
 		MvcResult rtn = mockMvc.perform(get("/publication/deleted").accept(MediaType.APPLICATION_JSON))
 				.andExpect(status().isOk())
-				.andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8_VALUE))
+				.andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8))
```

Comment:
I'm not sure that this matches the actual content type delivered by the real service. These "integration" tests may need more massaging to better reflect the real output of the service. This may also require adding UTF-8 data to the tests.